modals/src/main/res/values/styles-modals.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:ns2="http://schemas.android.com/tools">
+
+    <style name="ThemeOverlay.Vitamin.MaterialAlertDialog" parent="ThemeOverlay.MaterialComponents.MaterialAlertDialog">
+        <item name="colorSecondary">?attr/vtmnContentAction</item>
+        <item name="colorSurface">?attr/vtmnBackgroundPrimary</item>
+        <item name="colorOnSurface">?attr/vtmnContentSecondary</item>
+        <item name="materialAlertDialogTitleTextStyle">@style/MaterialAlertDialog.Vitamin.Title.Text</item>
+        <item name="materialAlertDialogTitleIconStyle">@style/MaterialAlertDialog.Vitamin.Title.Icon</item>
+        <item name="materialAlertDialogBodyTextStyle">@style/MaterialAlertDialog.Vitamin.Body.Text</item>
+        <item name="android:checkedTextViewStyle">@style/MaterialAlertDialog.Vitamin.Body.Text</item>
+        <item name="buttonBarPositiveButtonStyle">@style/Widget.Vitamin.Button.Ghost.Modal</item>
+        <item name="buttonBarNegativeButtonStyle">@style/Widget.Vitamin.Button.Ghost.Modal</item>
+        <item name="buttonBarNeutralButtonStyle">@style/Widget.Vitamin.Button.Ghost.Modal</item>
+    </style>
+
+    <style name="ThemeOverlay.Vitamin.MaterialAlertDialog.Centered" parent="ThemeOverlay.Vitamin.MaterialAlertDialog">
+        <item name="materialAlertDialogButtonSpacerVisibility">@integer/mtrl_view_gone</item>
+        <item name="buttonBarPositiveButtonStyle">@style/Widget.Vitamin.Button.Ghost.Modal.Centered</item>
+        <item name="buttonBarNegativeButtonStyle">@style/Widget.Vitamin.Button.Ghost.Modal.Centered</item>
+        <item name="buttonBarNeutralButtonStyle">@style/Widget.Vitamin.Button.Ghost.Modal.Centered</item>
+    </style>
+
+    <style name="MaterialAlertDialog.Vitamin.Title.Text" parent="MaterialAlertDialog.MaterialComponents.Title.Text">
+        <item name="android:textAppearance">?attr/textAppearanceSubtitle1</item>
+        <item name="android:textColor">?attr/vtmnContentPrimary</item>
+    </style>
+
+    <style name="MaterialAlertDialog.Vitamin.Title.Icon" parent="MaterialAlertDialog.MaterialComponents.Title.Icon">
+        <item name="android:layout_marginEnd" ns2:ignore="NewApi">8dip</item>
```

Comment:
I'm curious to have info about this : 
- why a `tools:ignore` here
- why called it `ns2`
- Why `dip` and not `dp` ?
- Why 8 and not 9 as below ?